cocoa/tests_backend/widgets/webview.py

```diff
@@ -0,0 +1,10 @@
+from toga_cocoa.libs import WKWebView
+
+from .base import SimpleProbe
+
+
+class WebViewProbe(SimpleProbe):
+    native_class = WKWebView
+
+    async def get_page_content(self):
+        return await self.impl.evaluate_javascript("document.body.innerHTML")
```

Comment:
If this method is going to be the same on all platforms, why not just make the test call `widget.evaluate_javascript`?

Reply:
Wrapping the API makes it a little more explicit what the functionally doing, rather than how it's implemented; but if you don't think it's worth it, I don't mind this being simplified.